src/test/java/stringCalculator/CustomDelimiterCondition.java

```diff
@@ -0,0 +1,19 @@
+package stringCalculator;
+
+public class CustomDelimiterCondition {
+        final private String start;
+        final private String end;
```

Comment:
이 클래스의 역할과 책임은 무엇일까요? 🤔

Reply:
```suggestion

public class CustomDelimiterCondition {
    private final String start;
    private final String end;
```

```suggestion

public record CustomDelimiterCondition(String start, String end) {
}
```

record는 봐도 봐도 저도 익숙하지는 않네요. 🥲

Reply:
사실상 행동과 책임이 없는 단순 데이터 오브젝트인데 이런 경우 record 를 쓰나보군요.
자바는 익숙치 않아서 ..